ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml

```
@@ -72,6 +72,29 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: ocm-e2e-kind
+- as: test-e2e-grc-framework
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: grc-policy-framework-tests
+    env:
+      CLUSTERPOOL_GROUP_NAME: policy-grc
+      CLUSTERPOOL_HOST_NAMESPACE: acm-grc-security
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: ocm-grc-clusterpool
+      CLUSTERPOOL_LIFETIME: 4h
+      CLUSTERPOOL_LIST_EXCLUSION_FILTER: dev\|autoclaims
+      CLUSTERPOOL_LIST_INCLUSION_FILTER: prow
```

Normally this workflow patches the RHACM component to match the PR, so this will prevent it from attempting to patch the `grc-policy-framework-tests` image (which doesn't exist so it'll probably fail):
```suggestion
      CLUSTERPOOL_LIST_INCLUSION_FILTER: prow
      SKIP_COMPONENT_INSTALL: true
```